src/services/WolfDen.Application/Validators/LeaveManagement/LeaveRequestValidator.cs

```diff
@@ -0,0 +1,16 @@
+﻿using FluentValidation;
+using WolfDen.Domain.Entity;
+
+namespace WolfDen.Application.Validators.LeaveManagement
```

Comment:
Write validators in the folder along with the its corresponding handler and  commands

Reply:
validators moved to respective folder containing command and command Handler